ansible/roles/biocache-hub/templates/config/config.properties

```diff
@@ -14,7 +14,7 @@ security.cas.casServerLogoutUrl={{ auth_base_url }}/cas/logout
 security.cas.loginUrl={{ auth_base_url }}/cas/login
 security.cas.logoutUrl={{ auth_base_url }}/cas/logout
 security.cas.uriFilterPattern={{ biocache_hub_uri_filter_pattern | default('/admin/*,/alaAdmin/*,/download/*') }}
-security.cas.authenticateOnlyIfLoggedInFilterPattern={{ biocache_hub_authenticate_only_if_logged_in_filter_pattern | default('/occurrences/*,/explore/your-area,/query,/proxy/download/*,/') }}
+security.cas.authenticateOnlyIfLoggedInFilterPattern={{ biocache_hub_authenticate_only_if_logged_in_filter_pattern | default('/query,/proxy/download/*') }}
```

Comment:
The removal of /occurrences/* breaks the role conditional content of occurrence pages, i.e. club view.